Tests/SwiftDocCTests/Converter/RenderNodeCodableTests.swift

```diff
@@ -72,7 +72,23 @@ class RenderNodeCodableTests: XCTestCase {
             XCTFail(error.localizedDescription)
         }
     }
-    
+
+    func testSortedKeys() {
+        var renderNode = bareRenderNode
+        renderNode.metadata.title = "title"
+        renderNode.metadata.category = "category"
+
+        do {
+            let encoder = RenderJSONEncoder.makeEncoder()
+            let encodedData = try renderNode.encodeToJSON(with: encoder)
+            let jsonString = String(data: encodedData, encoding: .utf8)!
+            let expectedSubstring = #"{"category":"category","title":"title"}"#
```

Comment:
I think this test would need a similar `#available` check in order to not fail on platforms and versions that don't support sorted JSON keys.